proxy/http/HttpSM.h

```diff
@@ -519,6 +522,18 @@ class HttpSM : public Continuation, public PluginUserArgs<TS_USER_ARGS_TXN>
   /// Update the milestones to track time spent in the plugin API.
   void milestone_update_api_time();
 
+  sockaddr *
+  get_server_ip() const
```

Comment:
There are two IP addresses associated with the server, the method name should indicate which one. The standard terms are "local" (address used on the ATS end of the socket) and "remote" (address used on the non-ATS end of the socket). This should probably be `get_server_remote_addr`.